README.md

```diff
@@ -141,36 +154,41 @@ yarn start # start the dev server
 
 A development server should now be running on [localhost:4000](localhost:4000).
 
-## Step 2. View your Blocks within a sandbox
+## Step 2. View your development server within the Blocks app
 
-When you visit [localhost:4000](localhost:4000) in your browser, you'll see an interface where you can test out your Block.
+When you visit [localhost:4000](localhost:4000) in your browser, you'll be
+redirected to the Blocks app with your locally-developed blocks embedded in it.
+You can see your blocks by clicking on the block picker toward the top of the
+page (they're shown at the top in blue with a plug icon):
 
-This starter project has one example folder block and one example file block.
+<img src="https://user-images.githubusercontent.com/56439/181648955-101b6567-3f9b-44b3-af99-7ef3ca6161b9.png" width="418" />
```

Comment:
we might want alt tags for images/videos

Reply:
updated alts on all images 🕺